README.md

```diff
@@ -1 +1,144 @@
-# bookstore
\ No newline at end of file
+<a name="readme-top"></a>
+
+  <h3><b>BookStore 2.0: README</b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Math Magicians <a name="about-project"></a>
+
+**BookStore2.0** is an innovative and visually stunning web application that redefines the way we experience and interact with books.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **N/A**
+- **N/A**
+- **N/A**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Nodejs
+- A text editor of your choice
+- A properly working pc or laptop
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd my-folder
+  git clone https://github.com/naolkidanu/bookstore.git
+```
+
+### Install
+
+Install this project with node packages:
+
+```sh
+  cd my-project
+  npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+```sh
+  npm jest
+```
+
+### Deployment
+
+You can deploy this project using: GitHub but if you want to deploy make sure you let me know first :3
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
```

Comment:
- [ ] Kindly remove the `section` comment, because this comment is only for reference for the project author not for the public to read. Leaving such kind of comment confuses any person and also looks unprofessional.

Reply:
Hello @skyv26 
I appreciate your feedback, and I'll correct it.

Reply:
This change has been done well